brunoise/scanning.py

```diff
@@ -183,11 +199,12 @@ def scan_loop(self, read_task, write_task):
                     number_of_samples_per_channel=self.n_samples_in,
                     timeout=1,
                 )
+                self.time_queue.put(perf_counter())
```

Comment:
@ywkure do you want to keep this? After #25 is merged it should not be needed anymore. But it wouldn't hurt either, so you're choice. :)

Reply:
I think we can keep it for now.